src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,32 @@
+.icon {
+  display: block;
+  height: 24px;
+  width: 24px;
+  background-image: url(../images/icons/icon-burger-menu.svg);
+  background-size: cover;
+  transition: transform $effectDuration;
+
+  @include hover(transform, scale(1.2));
+
+  @include on-tablet {
+    height: 32px;
+    width: 32px;
+  }
+
+  &--phone {
+    background-image: url(../images/icons/icon-phone-Call.svg);
+  }
+
+  &--menu {
+    background-image: url(../images/icons/icon-burger-menu.svg);
```

Comment:
**Inconsistent Case in File Names:**

CSS URLs are case-sensitive. For example:
icon-burger-menu.svg is lowercase, while later you reference Icon-Burger-menu.svg with mixed case. If your actual file names don't match this case exactly, the images won't load.

https://developer.mozilla.org/en-US/docs/Web/CSS/background-image